index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link type="text/css" rel="stylesheet" href="./Assets/Styles/index.css">
+    <title>Document</title>
+</head>
+<body id="body">
+    <div id="mobile_navbar" class="mobile_navbar">
+        <img id="hamburger_btn" class="hamburger_btn" alt="menu" src="Assets/Images/Icons/hamburger.png">
+    </div>
+    <nav class="desktop_navbar">
+            <ul class="dt_navbartop">
+                <li class="dt_nav_bar_item"><img class="dt_facebook-link" alt="facebook" src="./Assets/Images/Icons/facebook.png"></li>
+                <li class="dt_nav_bar_item"><img class="dt_twitter-link" alt="twitter" src="./Assets/Images/Icons/twitter.png"></li>
+                <li class="dt_nav_bar_item"><a class="dt_english-link">English</a></li>
+                <li class="dt_nav_bar_item"><a class="dt_my-page-link">My Page</a></li>
+                <li class="dt_nav_bar_item_last"><a class="dt_logout-link">Logout</a></li>
+            </ul>
+            <ul class="dt_navbarbottom">
+                <li class="dt_bottom_bar_item_first"><a href="./index.html"><img class="item_first_img" alt="home" src="./Assets/Images/Logos/ifclogo.png"></a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="./about.html">About</a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="#sec_2">Program</a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="#sec_3">Speakers</a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="#sec_4">Sponsors</a></li>
+                <li class="dt_bottom_bar_item"><a class="top_link" href="#sec_5">Credits</a></li>
+                <li class="dt_bottom_bar_item_last"><a class="dt_cc-campaign-link">INT Food C.</a></li>
+            </ul>
+    </nav>
+    <div id="menu_display" class="menu_display">
+        <img id="close_menu_btn" class="close_menu_btn" alt="close" src="./Assets/Images/Icons/close.png">
+        <ul class="md_links">
+            <li class="md-item"><a class="md-link-asside" href="./about.html">About</a></li>
+            <li class="md-item"><a class="md-link" href="#sec_2">Main Program</a></li>
+            <li class="md-item"><a class="md-link" href="#sec_3">Featured Speakers</a></li>
+        </ul>
+    </div>
+    <section id="sec_1" class="sec_1">
+        <h1 class="sec_1_greetings">"Hello! Chefs community"</h1>
```

Comment:
* This line of text (hello chefs community) is not  visible, please  adjust it 🙏 
![Screenshot (123)](https://user-images.githubusercontent.com/52497246/124194328-4ae48b80-dac0-11eb-8510-680a93f934e8.png)

Reply:
* Good job making the requested changes but i can see your middle section on mobile view has some issues 
* YOURS:
![Screenshot (127)](https://user-images.githubusercontent.com/52497246/124205244-c2261980-dad8-11eb-828e-15d9a89e83d8.png)
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ORIGINAL :
![Screenshot (128)](https://user-images.githubusercontent.com/52497246/124205291-df5ae800-dad8-11eb-85d6-62cc0b80cbc9.png)